docs/reference/template-tags.md

```diff
@@ -0,0 +1,200 @@
+# Template tags
+
+Apostrophe template tags add additional functionality to templates, such as inserting widget areas, async components, and template fragments. The tags described below are specific to Apostrophe, though they use the standard Nunjucks syntax: `{% tagName %}`. See standard tags in the [Nunjucks reference](https://mozilla.github.io/nunjucks/templating.html#tags).
+
+If a template tag takes multiple arguments they will be comma-separated. Additional context data may be included after a `with` keyword. See `area` below for examples of both.
+
+| Tag name | Description | Self-closing |
+| -------- | ----------- | ------------ |
+| [`area`](#area) | Insert a [widget area](/guide/areas-and-widgets.md) | Yes |
+| [`component`](#component) | Insert an [async component](/guide/async-components.md) | Yes |
+| [`fragment`](#fragment) | Declare a [template fragment](/guide/fragments.md) | No |
+| [`render`](#render) | Insert a basic template fragment | Yes |
+| [`rendercall`](#rendercall) | Insert a template fragment that includes a `rendercaller` slot | No |
+| [`widget`](#widget) | Used in the core area template to render individual widgets | Yes |
+
+
+## `area`
+
+The `area` tag inserts an area field into the template. The area field [must already be configured](/guide/areas-and-widgets.md#basic-area-configuration) in the page.
+
+### Usage
+
+```django
+{% area context, areaName with contextOptions %}
+```
+
+**Example:**
+```django
+{# Without context options (most typical) #}
+{% area data.page, 'sidebar' %}
+
+{# Including context options #}
+{% area data.page, 'main' with {
+  '@apostrophecms/image': {
+    sizes: '(min-width: 600px) 45vw, (min-width: 1140px) 530px'
+  }
+} %}
+```
+
+### Arguments
+
+#### `context`
+
+The area's document context, either a page (`data.page`), piece (`data.piece`), or widget (`data.widget`). The area field must be defined in the field schema for that context. See the [template data](/guide/template-data.md) section for more on each `data` property.
+
+#### `areaName`
+
+The name (a string) of the area field as defined in the field schema.
+
+#### `contextOptions` (optional)
+
+The context options object is added after area tag arguments following the `with` keyword. It is an object with keys matching the names of widget types allowed in that particular area field. Each key is assigned a value that will be passed into the widget template as `data.contextOptions`. See the [image widget guide](/guide/core-widgets.md#image-widget) for an example.
+
+Context options are optional for all core and official Apostrophe widget types.
+
+::: note
+Context options are not the best place for most widget configuration. That should be done in the [area field configuration](/reference/field-types/area.md#widgets). Context options are used to override that with options that only apply to the specific template context.
```

Comment:
I would say "supplement," not "override." You can't override anything. (OK technically if a particular widget.html is specifically written to look in one place before the other you could "override" in that sense, but we can't suggest in any way that you can override the widget schema of the area or people will keep trying to do that 😂)

So I would change "override" to "supplement" and add this sentence:

"Context options cannot  change which widgets are permitted in an area."